docs/_static/nidcpower_usage.inc

```diff
@@ -22,10 +22,10 @@ The following is a basic example of using the **nidcpower** module to open a ses
         row_format = '{0:3d}:   {1:8.6f}   {2:8.6f}   {3}'
         with session.initiate():
             while samples_acquired < 20:
```

Comment:
I think you can simplify the example and just fetch all `session.measure_record_length` values. The way we have it is a little odd because if fetch is called twice, the count will start from 0 somewhere in the middle.

So I would get rid of the while loop (assume timeout is large enough).